ui/lib/core/addon/utils/key-codes.js

```diff
@@ -3,6 +3,13 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
+/*
+ * DEPRCATED (see: https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode).
```

Comment:
Good catch, if we don't already can you make a ticket for this and throw it in our backlog or add to sustaining?